Documentation/quickstart.md

```diff
@@ -20,9 +24,19 @@ To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [fo
 
 If you are using `dataDirHostPath` to persist rook data on kubernetes hosts, make sure your host has at least 5GB of space available on the specified path.
 
-## Deploy Rook
+## TL;DR
```

Comment:
Instead of TL:DR, how about the title "I'm Feeling Lucky"? 
I saw this title in some docs and i thought it was clever and  funny :)

Reply:
The "TL;DR" is used for Helm Charts in the

Reply:
Instead of changing the title, what about starting the paragraph with "If you're feeling lucky, a simple Rook cluster can be created with the following `kubectl` commands. For the more detailed install, skip to the next section to deploy the Rook operator."

Reply:
@travisn You are missing the point. I really like that title. It is all about the title. :P

Reply:
i don't think i'm ready for that title :)